apps/address-service/domains/common/utils/services/search/providers/GoogleSearchProvider.js

```diff
@@ -0,0 +1,236 @@
+const { AbstractSearchProvider } = require('./AbstractSearchProvider')
+const { GOOGLE_PROVIDER } = require('@address-service/domains/common/constants/providers')
+const conf = require('@open-condo/config')
+const get = require('lodash/get')
+const fetch = require('node-fetch')
+const { DADATA_PROVIDER } = require('../../../../constants/providers')
```

Comment:
I think it's time to make a linter rule for this)

Reply:
fixed